src/compressed_tensors/compressors/quantized_compressors/base.py

```diff
@@ -137,6 +151,21 @@ def decompress(
                 full_name = merge_names(weight_name, param_name)
                 with safe_open(safe_path, framework="pt", device=device) as f:
                     weight_data[param_name] = f.get_tensor(full_name)
+            if "weight_scale" in weight_data:
```

Comment:
should weight_scale be hardcoded regardless of the compressor ?

Reply:
Yes, this is how it was done before as well, which seems fine to me since compress also has knowledge of weight_scale

Reply:
@horheynm the presence of the scale is the way we determine whether or not something was quantized